src/libwalletqt/WalletManager.h

```diff
@@ -185,14 +186,14 @@ class WalletManager : public QObject
     // clear/rename wallet cache
     Q_INVOKABLE bool clearWalletCache(const QString &fileName) const;
 
-    Q_INVOKABLE void onWalletPassphraseNeeded(Monero::Wallet * wallet);
-    Q_INVOKABLE void onPassphraseEntered(const QString &passphrase, bool entry_abort=false);
+    Q_INVOKABLE void onPassphraseEntered(const QString &passphrase, bool enter_on_device, bool entry_abort=false);
+    Q_INVOKABLE virtual void onWalletPassphraseNeeded(bool on_device) override;
```

Comment:
This is not getting invoked from QML too

Reply:
done